crypto/aes/aes_x86core.c

```diff
@@ -80,17 +80,6 @@ static void prefetch256(const void *table)
 #undef GETU32
 #define GETU32(p) (*((u32*)(p)))
 
-#if (defined(_WIN32) || defined(_WIN64)) && !defined(__MINGW32__)
```

Comment:
I think I added this for originally to fix a problem with AIX truncation..
These defines look like they were copied from modes.h?
Where does the U64() come from with this change?